numpy/core/include/numpy/npy_3kcompat.h

```diff
@@ -13,11 +13,9 @@
 #include <Python.h>
 #include <stdio.h>
 
-#if PY_VERSION_HEX >= 0x03000000
 #ifndef NPY_PY3K
 #define NPY_PY3K 1
```

Comment:
I removing the internal use of this is one of the next steps.

Reply:
Yes, I'm also working on cleaning that up (in another CL)